src/components/common/notification.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import { Button } from 'attractions';
+  import getNotificationContent from '@/constants/notification-content.js';
+  import { formatTime } from '@/utils/date-time-format.js';
+
+  export let notification;
+
+  const dispatch = createEventDispatcher();
+</script>
+
+<li class="notification">
+  <span class="content">
+    {#each getNotificationContent(notification) as fragment}
+      {#if typeof fragment === 'string'}
+        {fragment}
+      {:else if fragment.bold}
+        <strong>{fragment.text}</strong>
+      {:else}
+        <a href="{fragment.url}" rel="prefetch">{fragment.text}</a>
+      {/if}
+    {/each}
+  </span>
+  <Button round title="Mark as read" class="mt" on:click={() => dispatch('read', notification)}>
+    <svg src="images/icons/check.svg" class="icon" />
+  </Button>
+  <time datetime={notification.timestamp}>{formatTime(notification.timestamp)}</time>
+</li>
+
+<style src="../../../static/css/components/common/notification.scss"></style>
```

Comment:
Does the `@` alias not work here?

Reply:
It probably does, but it's pointing to /src/ anyway, so it wouldn't be of much use. Of course, we could devise another alias, but honestly I'm thinking of removing that aliasing plugin. I just don't like it. And now that we got a lot of components from Attractions, having the aliasing for, say, `/ui/` is not that critical.

Reply:
Do you not like the `@` alias or the idea of aliasing in general? I think it is helpful in that we don't need to write a lot of `../../../`, and especially when moving a file to a different folder. If your problem is with the `@` symbol, it can be changed to any other symbol/word

Reply:
The idea of aliasing in general. As I said, I was tearing hairs from my head when I first opened the codebase because I couldn't figure anything out, and only then I asked Max and he pointed me at the rollup config. I believe no saved keystrokes are worth the huge drawback on clarity

Reply:
Even if the alias was something like `src`? I'd think it seems pretty clear

Reply:
More than anything, I'd desire a `/` alias, everything else is manageable. But I believe this is not possible.

Reply:
Why note? There doesn't seem to be a mention of such limitation in [the docs](https://github.com/rollup/plugins/tree/master/packages/alias), and the source code doesn't suggest any problems with a `/` alias. Needs to be tested

Reply:
Okay, I tried everything I could, apparently, it's that difficult to have Rollup import from project root. I couldn't get it to work, you're free to give it a shot. Note that `<style src="...">` might be resolved without Rollup, simply with Node, but that's not certain. idk how Rollup takes over file resolution.